### PR TITLE
Generate SDG topics MCF.

### PR DESCRIPTION
**Pull request by keyurva**
* See the generated MCF [here][cns].
  + Will create a CL for g3 after this PR is submitted.
* @pradh - This commit includes the updated topic cache JSON. Let me know if that should be removed.

[cns]: https://cnsviewer-static.corp.google.com/cns/jv-d/home/datcom/keyurs/custom_topics_sdg.mcf?user=

### CODE REVIEW

File: tools/sdg/topics/main.py [hunk 1]
```diff
@@ -161,6 +163,25 @@ def generate_full():
...
+      out.write(generate_mcf_node(node))
+
+
+def generate_mcf_node(node: dict) -> str:
+  return f"""Node: dcid:{node['dcid'][0]}
+dcid: "{node['dcid'][0]}"
+relevantVariable: "{','.join(node['relevantVariableList'])}"
+name: "{node['name'][0]}"
+typeOf: dcid:{node['typeOf'][0]}
+
```
Comment (pradh) [inline tools/sdg/topics/main.py:183]:
```
relevantVariable: "sdg/AG_FOOD_WST,dc/topic/sdgAGFOODWST_sdgfoodWasteSector"
```

```
relevantVariable: dcid:sdg/AG_FOOD_WST, dcid:dc/topic/sdgAGFOODWST_sdgfoodWasteSector
```

There's an important difference here --- first one is a single string, second one is a list of references!

Comment (keyurva) [reply tools/sdg/topics/main.py:183]: Thanks - still grokking MCF! Done. MCF in CNS is updated.

File: tools/sdg/topics/main.py [hunk 2]
```diff
@@ -161,6 +163,25 @@ def generate_full():
...
+  logging.info("Writing MCF to: %s", _MCF_PATH)
+  os.makedirs(_TMP_DIR, exist_ok=True)
+  with open(_MCF_PATH, 'w') as out:
+    for node in nodes:
+      out.write(generate_mcf_node(node))
+
+
+def generate_mcf_node(node: dict) -> str:
+  return f"""Node: dcid:{node['dcid'][0]}
+dcid: "{node['dcid'][0]}"
```
Comment (pradh) [inline tools/sdg/topics/main.py:179]: nit: we don't need the `dcid:` line... its redundant when using 'Node: dcid:X'

https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#use-dcid-ref-as-node-name

**Review by pradh — APPROVED** [review]
Thanks!

File: tools/sdg/topics/main.py (hunk 2 above)
Comment (keyurva) [reply tools/sdg/topics/main.py]: Thanks. Removed.

**Pull request merged**